### PR TITLE
fix: fix tests and build failures and incorrect padding

### PR DESCRIPTION
**Pull request by kazo0**
fixes from canary failures

update winappsdk
fix padding improperly set on tabbar
fix failing tests

### CODE REVIEW

File: src/Uno.Toolkit.RuntimeTests/Tests/NavigationBarTests.cs [hunk 1]
```diff
@@ -23,6 +23,7 @@
 using Microsoft.UI.Xaml.Navigation;
 using Microsoft.UI.Xaml;
 using Microsoft.UI;
+using Windows.Networking.Connectivity;
```
Comment (Xiaoy312) [inline src/Uno.Toolkit.RuntimeTests/Tests/NavigationBarTests.cs:26]: no needs
```suggestion
```

File: src/Uno.Toolkit.RuntimeTests/Tests/NavigationBarTests.cs [hunk 2]
```diff
@@ -49,24 +50,27 @@ public async Task MainCommand_In_Popup_Without_Page(MainCommandMode mainCommandM
...
 				Assert.IsTrue(popup.IsOpen == !shouldGoBack, "Popup is in an incorrect state");
+			};
+
+			try
+			{
+				await UnitTestUIContentHelperEx.SetContentAndWait(content);
+
+				popup.Opened += popupOpened;
+				popup.IsOpen = true;
 			}
```
Comment (Xiaoy312) [inline src/Uno.Toolkit.RuntimeTests/Tests/NavigationBarTests.cs:70]: we are not waiting on the `Opened` callback

File: src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs [hunk 3]
```diff
@@ -229,18 +229,17 @@ public async Task Verify_Indicator_Transitions(Orientation orientation, Indicato
 
 			var rootGrid = XamlHelper.LoadXaml<Grid>(@$"
```
Comment (Xiaoy312) [inline src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs:230]: you can use $``` to avoid the "" and write like normal xaml (with the double-{} ofc)

Comment (Xiaoy312) [reply src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs:230]: also are the new colors meant to be kept?

Comment (kazo0) [reply src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs:230]: oh woops no the colors should be removed :)

**Review by francoistanguay — APPROVED** [review]

**Pull request merged**